src/main/scala/uk/gov/nationalarchives/tdr/api/service/CustomMetadataPropertiesService.scala

```diff
@@ -101,11 +99,3 @@ class CustomMetadataPropertiesService(customMetadataPropertiesRepository: Custom
     case _                => throw new Exception(s"Invalid property type $propertyType")
   }
 }
-
-object CustomMetadataPropertiesService {
```

Comment:
Is there a reason the companion object is deleted? Was this added when using the new `protectedProperty` field?

Reply:
Yep, there were added with the new protectedProperty field